tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "noEmit": true,
     "jsx": "react"
   },
-  "include": ["src"]
+  "include": ["src", "scripts/prepare_regions_data"]
```

Comment:
Heh.  Guessing this was temporary for development, until your other PR lands?

Reply:
I think my editor did that actually, but yeah, I'll remove it.